packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1711,6 +1712,40 @@ QUnit.module('options changed', moduleSetup, () => {
     });
 });
 
+QUnit.module('focusOnDeletion', () => {
+    const items = ['item 1', 'item 2', 'item 3'];
+    QUnit.module('focusStateEnabled', function(assert) {
+        QUnit.test('list item should not be focus when deleting when focusStateEnabled = false (T1226030)', function(assert) {
+            const $element = $('#list').dxList({
+                items,
+                focusStateEnabled: false,
+                allowItemDeleting: true,
+                itemDeleteMode: 'static',
+            });
+            let $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            $listItems.eq(0).find(`.${STATIC_DELETE_BUTTON_CLASS}`).trigger('dxclick');
+            $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            assert.notOk($listItems.eq(0).hasClass('dx-state-focused'), 'Next Item is not focused');
+        });
+
+        QUnit.test('list item should be focus when deleting when focusStateEnabled = true (T1226030)', function(assert) {
+            const $element = $('#list').dxList({
+                items,
+                allowItemDeleting: true,
+                itemDeleteMode: 'static',
+            });
+            let $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            $listItems.eq(0).find(`.${STATIC_DELETE_BUTTON_CLASS}`).trigger('dxclick');
+            $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            assert.ok($listItems.eq(0).hasClass('dx-state-focused'), 'Next Item is focused');
```

Comment:
so, we don't have a string constant with "dx-state-focused" value as in other files? Than let's create it and refactor similar places in this file! 
[Boy Scoute Rule](https://innolution.com/resources/glossary/boy-scout-rule#:~:text=If%20you%20find%20a%20mess,messier%2C%20than%20you%20found%20it.)

Reply:
[b8bba63](https://github.com/DevExpress/DevExtreme/pull/27230/commits/b8bba636980d2478c625678291fb31d98286ed70)